### ISSUE TITLE
Impossible d'installer le plugin sur QGIS

### ISSUE THREAD

**Issue by evinouze** [geo2france/idg-qgis-plugin#63]
Bonjour,

# Diagnostic
Lorsque je tente d'installer le plugin "IDG" depuis QGIS, j'ai le message d'erreur suivant qui s'affiche : 

> Error transferring https://plugins.qgis.org/plugins/idg/version/0.2.4/download/ - server replied: Bad Gateway

![image](https://github.com/geo2france/idg-qgis-plugin/assets/26651892/aae2e203-4761-45eb-bf06-1cdacc77d9fd)

Pourriez-vous m'indiquer si vous reproduisez la même erreur ?

# Contexte

Version de QGIS | 3.28.9-Firenze | Révision du code | 66f4d63a
-- | -- | -- | --
Version de Qt | 5.15.8
Version de Python | 3.9.9
Version de GDAL/OGR | 3.5.2
Version de Proj | 8.2.1
Version de la base de données du registre EPSG | v10.041 (2021-12-03)
Version de GEOS | 3.11.2-CAPI-1.17.2
Version de SQLite | 3.36.0
Version de PDAL | 2.4.3
Version du client PostgreSQL | unknown
Version de SpatiaLite | 5.0.1
Version de QWT | 6.1.5
Version de QScintilla2 | 2.11.6
Version de l'OS | KDE Flatpak runtime
  |   |   |  
Extensions Python actives
changeDataSource | 3.1
craig | 0.3
StreetView | 3.2
QuickOSM | 2.2.3
menu_from_project | 2.1.0
french_address | 1.3.0
db_manager | 0.1.20
processing | 2.12.99

</body></html>

**Comment by lecault** [comment]
Bonjour,

Je n'ai pas de soucis.

Avez vous essayé d'installer une autre extension depuis le dépôt QGIS ?
Je suis en QGIS 3.28.8.

**Comment by evinouze** [comment]
> Bonjour,
> 
> Je n'ai pas de soucis.
> 
> Avez vous essayé d'installer une autre extension depuis le dépôt QGIS ? Je suis en QGIS 3.28.8.

Oui, dans la partie contexte, vpus pouvez voir que j'ai installé plusieurs extensions.

**Comment by jbdesbas** [comment]
@evinouze Est-ce que le lien https://plugins.qgis.org/plugins/idg/version/0.2.4/download/ fonctionne dans votre navigateur ?

**Comment by evinouze** [comment]
Oui, ca télécharge bien le fichier zip "idg-0.2.4.zip"
J'ai testé de nouveau dans QGIS et là j'arrive bien à installer le plugin :-)

**Comment by jbdesbas** [comment]
Ok, probablement une indispo temporaire de plugins.qgis.org 👍
